frontend/app/utils/locale-utils.ts

```diff
@@ -120,6 +120,6 @@ export async function switchLanguageCookie<T extends SwitchLanguageData['languag
  * @param { english translation, french translation}
  * @returns either the english translation or the french translation.
  */
-export function getNameByLanguage(language: string, { nameEn, nameFr }: { nameEn?: string; nameFr?: string }) {
-  return language == 'fr' ? nameFr : nameEn;
+export function getNameByLanguage(language: string, languageTranslation: { nameEn?: string; nameFr?: string } | null) {
```

Comment:
`| null` remove the null type. Null check should be made at when calling `getNameByLanguage`.

Reply:
Fixed it to be able to close the user story.